lightningd/bitcoind.c

```diff
@@ -814,12 +814,73 @@ static void fatal_bitcoind_failure(struct bitcoind *bitcoind, const char *error_
 	exit(1);
 }
 
+static char* get_blockchainname_for_cli(struct bitcoind *bitcoind)
```

Comment:
Actually, sorry for the churn -- would it not be better to instead have a `getblockchaininfo_chain` field in `struct chainparams`?

That way if new altcoins want to commit to maintaining a branch of C-lightning with support for their altcoin, they just need to fill in that field according to the `chain` param of their `getblockchaininfo` command.

Reply:
Thank you :+1: 
What about `bip70_name`?

Reply:
I am uncertain how [BIP-70 Payment Protocol](https://github.com/bitcoin/bips/blob/master/bip-0070.mediawiki) relates?